### ISSUE TITLE
[FEATURE REQUEST] Add list timelines as columns

### ISSUE THREAD

**Issue by f3rcho4r** [TwidereProject/TwidereX-Android#295]
In the old Twidere, any list's timeline could be added as an individual column to the main view. Said column would be treated as any timeline, with the possibility to read from bottom and even filter retweets out.

 The current 'List column' implantation is far from being as useful, since it only works as a shortcut to the lists section, with no added features.

**Comment by alphanerdgen** [comment]
Yes, we need this badly. Please dev, make it a priority in your TwidereX roadmap.

**Comment by umbellipher** [comment]
Just found this app (on F-droid) because of the Android API issues. Excellent - it still works.

+1 for this, please. Lists in columns would be great.
